cmd/bootstrap/bootstrap.go

```diff
@@ -77,72 +141,142 @@ func main() {
 			os.Exit(1)
 		}
 	}
-	// TODO defined standard places for input path?
+	namespace := userNamespace
+	if namespace == "" {
+		namespace = "default"
+	}
 	if inputPath == "" {
-		fmt.Printf("No input path specified\n")
+		// when input path is empty, but a namespace is provided, try to reload an existing site definition
+		existingPath := api.GetInternalOutputPath(namespace, api.LoadedSiteStatePath)
+		if _, err := os.Stat(existingPath); err == nil {
+			inputPath = existingPath
+			fmt.Printf("Sources will consumed from namespace %q\n", namespace)
+		} else {
+			fmt.Printf("Input path has not been provided and namespace %s does not exist\n", namespace)
+			os.Exit(1)
+		}
+	}
+
+	// if namespace already exists, fail if force is not set
+	_, err := os.Stat(api.GetInternalOutputPath(namespace, api.RuntimeSiteStatePath))
+	if !isBundle && err == nil && !*force {
+		fmt.Printf("Namespace already exists: %s\n", namespace)
 		os.Exit(1)
 	}
 
-	siteState, err := bootstrap(inputPath)
+	siteState, err := bootstrap(inputPath, userNamespace, internalbundle.GetBundleStrategy(bundleStrategy))
 	if err != nil {
 		fmt.Println("Failed to bootstrap:", err)
 		os.Exit(1)
 	}
 	var bundleSuffix string
-	if platform.IsBundle() {
+	if isBundle {
 		bundleSuffix = " (as a distributable bundle)"
+	} else {
+		bundleSuffix = fmt.Sprintf(" on namespace %q", siteState.GetNamespace())
+		// create bootstrap.out file
+		if api.IsRunningInContainer() {
+			outFile, err := os.Stat("/bootstrap.out")
+			if err == nil && !outFile.IsDir() {
+				err = os.WriteFile("/bootstrap.out", []byte(siteState.GetNamespace()), 0644)
+				if err != nil {
+					fmt.Println("Failed to write to bootstrap.out:", err)
```

Comment:
is this just informational? or should we do something (like exit) here?

Reply:
Actually, the message is not clear and it needs to be improved.

Basically if the file cannot be written for any reason, the bootstrap.sh script
won't be able to create the systemd service on the host machine, so this needs
to be explained, instead of only showing the error message.